frontend/src/homepage/ActiveStreamCard.jsx

```diff
@@ -14,10 +14,10 @@ export default function ActiveStreamCard() {
     <DashboardCard title="Active Streams" style={{ height: "100%" }}>
       <Grid container justify="center" direction="row" spacing={3}>
         <Grid item xs={12}>
-          <StreamsTableWrapper dataSource={dataSource} />
+          <StreamsTableWrapper dataSource={dataSource} isSimple />
```

Comment:
I'm worried there will be code duplication detected between SimpleStreamTable and FullStreamTable, which is what happened early on in this refactor when I tried that approach (without passing the columns as a prop but instead just using an isSimple boolean to determine which component to render). I know that this might be an antipattern as mentioned by @felixlapierre but there might also be the change of code duplication detected by SonarCloud, which I'm not sure if it's any better.